supplementary_style_guide/glossary_terms_conventions/general_conventions/v.adoc

```diff
@@ -69,7 +69,7 @@ Use lowercase "v" and uppercase "CPU".
 [discrete]
 [[vim]]
 ==== image:images/yes.png[yes] Vim (noun)
-*Description*: "Vim" is an acronym for Vi IMproved. In the original 1991 release for the Amiga platform, the acronym was derived from Vi IMitation. It became Vi IMproved when ported to various Unix-based operating systems in 1992. Despite being an acronym, and despite the first word of the "About" text that is displayed when you launch the editor, the standard, proper noun-derived, mixed-case spelling has been in use since its release on the Amiga.
+*Description*: "Vim" is an acronym for Vi IMproved. In the original 1991 release for the Amiga platform, the acronym was derived from Vi IMitation. It became Vi IMproved in 1992, when ported to the UNIX system and similar operating systems. Despite being an acronym, and despite the first word of the "About" text that is displayed when you launch the editor, the standard, proper noun-derived, mixed-case spelling has been in use since its release on the Amiga.
```

Comment:
I think one **despite** in the sentence, something like:

Despite being an acronym, and  the first word of the "About" text that is displayed when you launch the editor,

Reply:
Will take this up in a new PR.